app/models/miq_report/formatting.rb

```diff
@@ -95,6 +95,15 @@ def format(col, value, options = {})
     end
 
     options[:column] = col
+
+    # Chargeback Reports: Add the selected currency in the assigned rate to options
+    if db.to_s == "ChargebackVm" || db.to_s == "ChargebackContainerProject"
+      compute_selected_rate = ChargebackRate.get_assignments(:compute)[0]
+      storage_selected_rate = ChargebackRate.get_assignments(:storage)[0]
+      selected_rate = compute_selected_rate.nil? ? storage_selected_rate : compute_selected_rate
```

Comment:
I think that this is not what we need. We have compute and storage metric and there are for some certian fields.
For storage there are fields beginning with "storage":
![screen shot 2016-05-09 at 10 57 45](https://cloud.githubusercontent.com/assets/14937244/15109212/93045bb2-15da-11e6-88f7-87573dd4bad9.png)
and for these fields should be used "Storage" rate as well as the currency

I did some proposal https://github.com/ManageIQ/manageiq/issues/8538 how to solve it.